macros/seed_existance_check.sql

```diff
@@ -0,0 +1,38 @@
+{#
+Copyright (c) 2023-present Snowplow Analytics Ltd. All rights reserved.
+This program is licensed to you under the Snowplow Personal and Academic License Version 1.0,
+and you may not use this file except in compliance with the Snowplow Personal and Academic License Version 1.0.
+You may obtain a copy of the Snowplow Personal and Academic License Version 1.0 at https://docs.snowplow.io/personal-and-academic-license-1.0/
+#}
+
+{% macro seed_existance_check() %}
+    {% if execute %}
+        {# Ensure that this check is only performed during 'run' or 'build' commands #}
+        {%- if flags.WHICH in ('run') -%}
+            {% for node in graph.nodes.values() | selectattr("resource_type", "equalto", "seed") %}
+
+                {# Convert schema and table names to uppercase to prevent case sensitivity issues #}
+                {% set schema = node.schema | upper %}
+                {% set table = node.alias | upper %}
+                
```

Comment:
This is less important if you use the dbt_utils macro, but this is only worth doing is you ALSO call `upper` on the column itself in the information schema in the warehouse, as different warehouses may return them in different cases.

Reply:
<img width="864" alt="image" src="https://github.com/snowplow/dbt-snowplow-unified/assets/8260415/605fb176-1f0d-4b69-9bb5-b201aa75909a">
Yup, postgres returns the columns as lowercase